src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/investigate-mobile-app-crash-report.mdx

```diff
@@ -60,21 +60,10 @@ Use any of New Relic's [standard UI functions](/docs/data-analysis/user-interfac
   </Collapser>
 
   <Collapser
-    id="interaction-trail"
-    title="Interaction trail"
+    id="Event-trail"
+    title="Event trail"
   >
-    The **Interaction trail** shows the timeline of [interactions](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#interaction) traced within the app, from the beginning of the user session to the moment of the crash. This trail can help you reproduce the user actions that led up to the crash exception.
-
-    The **Interaction trail** links to the associated interaction in Mobile monitoring's [**Interactions** page](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/interactions-page). Use this link to view data about healthy, non-crashing execution of the listed interactions.
-
-    <Callout variant="tip">
-      To view the last interaction that occurred before the crash, select the interaction name that appears immediately before the crash.
-
-      <img
-        title="Mobile Crash Report: Interaction trail"
-        alt="Mobile Crash Report: Interaction trail"
-        src={mobileStackTraceandInteractionTrail}
-      />
+    
     </Callout>
```

Comment:
The build failed because there's an orphan callout closing tag, but since it seems you'll remove the collapser itself, the issue will go away.

```suggestion
 
```